src/logcollector/read_audit.c

```diff
@@ -19,19 +19,19 @@ static void audit_send_msg(char **cache, int top, const char *file, int drop_it,
     int i;
     size_t n = 0;
     size_t z;
-    char message[OS_MAXSTR];
+    char message[OS_MAXSTR - OS_LOG_HEADER] = {0};
```

Comment:
Why the `- OS_LOG_HEADER` ?

Reply:
Because the **w_msg_hash_queues_push** function has a buffer inside it in which the buffer sent with a header will be concatenated. The maximum size of the header is defined by the tag.